src/ir/item.rs

```diff
@@ -277,11 +278,20 @@ impl Item {
                     TypeKind::Named(ref name, _) => {
                         return name.to_owned();
                     }
-                    _ => {}
-                }
-
-                ty.name().map(ToOwned::to_owned)
-                         .unwrap_or_else(|| format!("_bindgen_ty{}", self.id()))
+                    // We really codegen and use the inner type, so use an empty
+                    // base name so codegen doesn't get confused.
```

Comment:
The start of this sentence has some funky grammar that prevents me from understanding it.

Is it supposed to be

> We really *want* codegen *to* ~~and~~ use the inner type, ...

?